### PR TITLE
#161/UI/AnotherButtonsColors/Refactor

### LINKED ISSUE #223: #161/UI/AnotherColors/Refactor

### PR DESCRIPTION
**Pull request by KatarzynaChmielecka**
closes #223

### CODE REVIEW

**Review by MonikaKrella — COMMENTED** [review]
will be approved after merge development

File: src/Organisms/BurgerMenu/BurgerMenu.js
Comment (MonikaKrella) [inline src/Organisms/BurgerMenu/BurgerMenu.js:32]: "/:userId" must be dynamic:
-url when we came from role page:
image
-url when we came from burger menu:
image

File: src/Organisms/NavElement/NavElement.js
Comment (MonikaKrella) [inline src/Organisms/NavElement/NavElement.js:31]: it works nice!
add it also to /forgot and /in-progress

**Review by MonikaKrella — CHANGES_REQUESTED** [review]

File: src/Organisms/NavElement/NavElement.js
Comment (KatarzynaChmielecka) [reply src/Organisms/NavElement/NavElement.js:31]: If yes- done ;)

Comment (MonikaKrella) [reply src/Organisms/NavElement/NavElement.js:33]: exactly that, great!

**Pull request merged**
